### PR TITLE
tasks were implemented

### PR DESCRIPTION
**Pull request by Serhii-Soloshych**
- [DEMO LINK](https://Serhii-Soloshych.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Serhii-Soloshych.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
add paddings 
image

image

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Linsk are broken

**Comment by Serhii-Soloshych** [comment]
> Linsk are broken

Hi, I think that because i did some mess in my git branches. Could you help me to fix them? 
image

File: src/styles/blocks/product-card.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    color: #060b35;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/product-card.scss:20]: move all colors into variables

**Comment by ivanchuk-yurii** [comment]
don't close your prs

**Comment by Serhii-Soloshych** [comment]
> don't close your prs

I know. But there was unexpectable problem. I made some mess in my branches, and I couldn't fix it. I asked another mentor to help me, but the easiest way was to remove my pull request.

**Pull request closed without merging**